cluster/swarm/cluster.go

```diff
@@ -968,6 +968,31 @@ func (c *Cluster) BuildImage(buildContext io.Reader, buildImage *types.ImageBuil
 	return nil
 }
 
+// RefreshEngines refreshes all containers in the cluster (UCP-specific)
```

Comment:
Whoops, forgot to remove that comment. Done.

I'm not sure how useful this will be for other applications but it's nice to be able to say "give me a list of containers without using the cache" (or rather, "refresh the cache and then give the containers"). Having a magical name filter is kind of a weird way to do this, but the Docker API doesn't give us a lot of other options; we could use HTTP headers, but the Docker client libraries make it non-trivial to send specific headers for a single request.